docs/getting-started/products/yeth/overview.md

```diff
@@ -0,0 +1,84 @@
+
+This dynamic adjustment allows the pool to optimize risk and yield distribution among the LSTs. In practice, this means that as an LST performs well or gains popularity, its weight in the pool may increase, attracting more liquidity and providing better returns.
+
+If an LST underperforms or faces issues, its weight may decrease, reducing its impact on the overall pool performance. This weight adjustment mechanism helps maintain an optimal risk-adjusted yield for yETH users.
+
+**How boostraping (launch) works?**
+
+![image](https://github.com/MarcoWorms/yearn-devdocs/assets/7863230/b92613a6-42e6-4936-affa-4c77a395ceb3)
+
+The bootstrapping process aims to kickstart yETH first LSTs and liquidity. You can read all about each phase in our bootstrapping release [article](https://medium.com/@marcoworms/yeth-lsd-lobbying-season-is-now-open-ff8a2537402d).
+
+# Contracts & Roles
+
+| Name                        | Address                             |
+|-----------------------------|-------------------------------------|
+| yETH                        |                                     |
+| st-yETH                     |                                     |
+| Management                  |                                     |
+| Guardian                    |                                     |
+| Whitelister                 |                                     |
+| Treasury                    |     ychad.eth                       |
+| Rate Providers              |                                     |
+
+## Management Role
+
+Trusted addresses with privileged access for limited operations. Should eventually be replaced by a smart contract:
+
+- Can start a gradual weight change, as long as no weight change is active yet.
+- Can whitelist a new asset, which sets an initial weight, sets the rate provider, and requires an initial deposit. New assets can only be whitelisted if no weight change has been scheduled yet.
+- Can update the rate provider for every whitelisted asset.
+- Can update the staking contract.
+- Can set the pool swap fee.
+- Can set the tolerance range.
+- Can set the new management address.
+- Can set the new guardian and whitelister addresses.
+- Can trigger [`pause mode`](pause-mode).
+- Can trigger [`killed mode`](killed-mode).
+
+#### Pause mode
+
+> This mode is to be enabled in the event of extreme market conditions or suspicious LST minting behavior or oracle activity.
+
+- No user may deposit assets into the contract.
+- Users may only withdraw assets in a balanced manner.
+- Management or guardian can undo pause mode to resume normal operation.
+
+#### Killed mode
+
+> This mode is to be activated in the event of a LST depegs, such as a mint bug or a compromised oracle or a critical bug in the protocol. This can also be used as part of upgrading to a new version of yETH.
```

Comment:
```suggestion
> This mode is to be activated in the event of critical failures, whether in the protocol itself or in any of the underlying LST tokens that back it. This can also be used as part of migrating to a new version of the yETH protocol.
```